### ISSUE TITLE
can't work in this website

### ISSUE THREAD

**Issue by sojey** [ultimate-enable-right-click/help#5]
win 10 x64 newest
edge beta 版本 111.0.1661.38 (正式版本) beta (64 位)

https://www.yigujin.cn/5914.html
not work. please fix it. thank you.

**Comment by ultimate-enable-right-click** [comment]
Websites can set text highlight background color, and this website sets it to be transparent. There is no visual indication that you are selecting texts when you are dragging on texts, but it is still being selected under the hood, and you can still copy texts via right-clicking or Ctrl+V.

**Comment by sojey** [comment]
I get it, thanks for the explanation, but this technique is really disgusting. Can you make this plugin solve this kind of problem?

**Comment by ultimate-enable-right-click** [comment]
I agree that it is disgusting, implementing the feature would be trivial, but can't come up with a nice UI to toggle this feature atm.
